app/example_app/books.py

```diff
@@ -0,0 +1,112 @@
+from flask import render_template, request, redirect, flash
+from .. import db
+from ..models import  Book, Announcement, Event, Graphic, Borrow
+from . import example_app
+
+@example_app.route("/Employee")
```

Comment:
Endpointy raczej małymi literami.
```suggestion
@example_app.route("/employee")
```